releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -39,6 +39,12 @@ For more details on Telemetry, see [here](https://learn.microsoft.com/en-us/azur
 
 Added support for variant feature flags defined using [Microsoft Feature Management schema](https://github.com/microsoft/FeatureManagement/blob/c5fab16dbf1450dce0bbfe7c4207da735ff31916/Schema/FeatureManagement.v2.0.0.schema.json). Variants and telemetry can be declared using [Microsoft Feature Flag schema v2](https://github.com/microsoft/FeatureManagement/blob/c5fab16dbf1450dce0bbfe7c4207da735ff31916/Schema/FeatureFlag.v2.0.0.schema.json). Here is a [Sample](https://github.com/microsoft/FeatureManagement-Dotnet/blob/f47e188babea0a91488d2e6a0b2ab4c9405d0794/examples/VariantAndTelemetryDemo/appsettings.json#L12).
 
+#### Additional Changes
+
+* Optimized memory allocation for calls to `IsEnabledAsync` and `GetVariantAsync`.
```

Comment:
I thought it's not just memory. Do we have numbers in terms of much improvement?

Reply:
The goal was to reduce memory allocation. Memory is now always more performant than v3.

With speed it's not as straightforward. v4 in .net6 is still slower than v3, while v4 in .net8 is faster than v3. I chose to omit it since it wasn't as easily stated.

Reply:
.NET 6 is deprecated soon. What's the perf improvement in .net8?